### DISCUSSION TITLE
No benefits received after approval of github student

### DISCUSSION THREAD

**Discussion by ashwin-nyu** [community/community#139906] [GitHub Education]
### Hi everyone,

Hey Everyone,

I got an approval for Github student pro account, but **benefits are not received yet**. Can anyone help me out with this? it's been 2 days already. I'm an NYU student.

Best regards,
Ashwin

**Comment by davevad93** [comment]
Hi @ashwin-nyu ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ashwin-nyu was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
